src/script/interpreter.h

```diff
@@ -146,6 +146,16 @@ enum : uint32_t {
     SCRIPT_VERIFY_END_MARKER
 };
 
+/**
+ * Mandatory script verification flags that all new blocks must comply with for
+ * them to be valid. (but old blocks may not comply with) Currently just P2SH,
+ * but in the future other flags may be added.
+ *
+ * Failing one of these tests may trigger a DoS ban - see CheckInputScripts() for
+ * details.
+ */
+static const unsigned int MANDATORY_SCRIPT_VERIFY_FLAGS = SCRIPT_VERIFY_P2SH;
```

Comment:
Contrary to its comment, this is only used when validating mempool transactions, so is better placed in policy.h

Reply:
Cherry picked 461259c4ecc1e48d028eaea56061e72a6667ce4f from #26291